prebuild.js

```diff
@@ -17,45 +17,39 @@ function prebuild (opts, target, runtime, callback) {
   buildLog(buildLogMessage)
   var abi = getAbi(target, runtime)
   var tarPath = getTarPath(opts, abi)
-  fs.stat(tarPath, function (err, st) {
-    if (!err && !opts.force) {
-      buildLog(tarPath + ' exists, skipping build')
-      return callback(null, tarPath)
-    }
-    var tasks = [
-      function (cb) {
-        build(opts, target, function (err, filename) {
-          if (err) return cb(err)
-          cb(null, filename)
-        })
-      },
-      function (filename, cb) {
-        buildLog('Packing ' + filename + ' into ' + tarPath)
-        pack(filename, tarPath, function (err) {
-          if (err) return cb(err)
-          cb(null)
-        })
-      }
-    ]
+  if (fs.existsSync(tarPath) && !opts.force) {
```

Comment:
I don't think we should start using sync calls in async methods. It's a bad pattern.